src/docs/swagger.json

```diff
@@ -14,15 +14,24 @@
   },
```

Comment:
I can't see the endpoint of follow and unfollow

Reply:
``/profiles/:username/follow`` this is how the path is structured @nkpremices

Reply:
cool @truestbyheart